.env.production

```diff
@@ -0,0 +1,2 @@
+REACT_APP_URL=http://localhost:3000
+REACT_APP_API_URL=http://localhost:8000/api/v1
```

Comment:
I take it these will change when we have our heroku app?

Reply:
Yep!